packages/api/lambdas/bulk-operation.js

```diff
@@ -91,12 +69,88 @@ async function bulkGranule(payload) {
   }
 
   // Remove duplicate Granule IDs
+  // TODO: could we get unique IDs from the query directly?
```

Comment:
Yeah you could use the cardinality aggregation for this. Not sure if it's more trouble than its worth. I could take a crack at it or pair if you want.

Reply:
I'll probably leave it out for now just to keep the scope in check but it would be good to circle back on later